src/sentry/tasks/sentry_apps.py

```diff
@@ -297,16 +297,34 @@ def send_and_save_webhook_request(url, sentry_app, app_platform_event):
     org_id = app_platform_event.install.organization_id
     event = "{}.{}".format(app_platform_event.resource, app_platform_event.action)
 
+    slug = sentry_app.slug
+    if sentry_app.status == 2:
+        slug = "internal"
+    if sentry_app.status == 0:
+        slug = "unpublished"
+
     try:
         resp = safe_urlopen(
             url=url, data=app_platform_event.body, headers=app_platform_event.headers, timeout=5
         )
+
     except RequestException:
+        metrics.incr(
+            "integration-platform.http_response",
+            sample_rate=1.0,
+            tags={"status": "timeout", "integration": slug},
```

Comment:
@ceorourke It would be good to include the event as a tag here so we can group by different events if we want to.

Reply:
maybe name it `webhook_event` or `webhook_type` or something more telling